src/cashier/currency.es6

```diff
@@ -4,6 +4,7 @@ import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import html from 'text!cashier/currency.html';
 
+require(['common/util'])
 require(['text!cashier/currency.html']);
```

Comment:
@arnabk I think there's no need to import the html, although it was already there before I made the change, so I thought of not removing it.